src/Services/AbstractAccessor.php

```diff
@@ -53,7 +53,14 @@ protected function pretty_export($var, $indent = '')
      */
     abstract public function getConfiguration(): array;
 
-    public function writeConfiguration($treeStore)
+    /**
+     * @deprecated Will be removed with Pimcore 11
```

Comment:
might be converted to abstract
```suggestion
     * @deprecated
```